DatabaseBuddy.Core/Extender/NumericExtender.cs

```diff
@@ -49,9 +79,11 @@ public static int ToInt32Value(this object Value, int DefaultValue = 0)
             }
             catch { return DefaultValue; }
         }
-        public static long ToMegabyte(this long Value)
+        public static long ByteToMegabyte(this long Value)
             => (long)(Value / Math.Pow(10, 6));
-        public static long ToByte(this long Value)
+        public static long MegaByteToByte(this long Value)
             => (long)(Value * Math.Pow(10, 6));
+        public static double MegaByteToGigaByte(this long Value)
```

Comment:
Create a `ByteToGigaByte` method instead of mega byte to gigabyte so we get a basic unit of byte to work with.